src/lib/isMobilePhone.js

```diff
@@ -49,6 +49,7 @@ const phones = {
   'en-JM': /^(\+?876)?\d{7}$/,
   'en-KE': /^(\+?254|0)(7|1)\d{8}$/,
   'fr-CF': /^(\+?236| ?)(70|75|77|72|21|22)\d{6}$/,
+  'en-GM': /^(\+220| ?)(5[0-2]|5[8-9]|2|3|6|7|9)\d{5,6}$/,
```

Comment:
The regexp does not match your provided source as it is missing a few ranges and also would match too long (e.g. `+22052123456` -> NSN Length 8 digits) and too short (e.g. `+220212345` -> NSN Length 5 digits) numbers as well.
Kindly check the national numbering plan below [1] and revise the regexp accordingly, thanks!

[1] https://www.itu.int/dms_pub/itu-t/oth/02/02/T020200004F0006PDFE.pdf / Page 2 + 3
alternative archived link:
https://web.archive.org/web/20230213231006/https://www.itu.int/dms_pub/itu-t/oth/02/02/T020200004F0006PDFE.pdf